jans-cedarling/cedarling_opa/convert.py

```
@@ -0,0 +1,226 @@
+            "The 'opa' binary was not found in your system PATH.\n"
+            "Please install it first: https://www.openpolicyagent.org/docs#install-and-run-opa"
+        )
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        temp_path = Path(temp_dir)
+        extract_dir = temp_path / "extracted"
+        Path.mkdir(extract_dir, exist_ok=True)
+
+        # 2. Extract input (.cjar/.zip or directory)
+        if input_path.is_file() and (
+            input_path.suffix == ".cjar" or input_path.suffix == ".zip"
```

Ruff (0.15.21)

[warning] Consider merging multiple comparisons: `input_path.suffix in {".cjar", ".zip"}`.

Merge multiple comparisons

(PLR1714)